specification/azurestackhci/AzureStackHCI.StackHCIVM.Management/main.tsp

```diff
@@ -48,7 +48,22 @@ enum Versions {
   /**
    * The 2024-05-01-preview API version.
    */
+  #suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-invalid-version-format" "Version name approved by reviewers"
```

Comment:
>"Version name approved by reviewers" [](http://example.com/codeflow?start=95&length=36)

can you give more details in a comment about this?

Reply:
This neednt be there. Removed it.